test/jdk/java/foreign/TestLayouts.java

```diff
@@ -51,6 +47,34 @@ public void testBadLayoutAlignment(MemoryLayout layout, long alignment) {
         layout.withBitAlignment(alignment);
     }
 
+    @Test
+    public void testNotEquals() {
+        List<MemoryLayout> basic = Stream.concat(Stream.of(basicLayouts), Stream.of(ADDRESS)).toList();
```

Comment:
Rather than using a stream and `forEach`, I suggest using a `@DataProvider` method, to stick with the theme in the existing tests. (That would also create a log line for each layout, so it's just a little bit easier to see which case(s) failed if there's a failure)